.github/workflows/e2e.v2.yaml

```
@@ -0,0 +1,184 @@
+            yq e ".spec.$FIELD=$TAG" -i ./.github/valdrelease/valdrelease.yaml
+          done
+      - name: Deploy Vald using Vald Helm Operator
+        if: ${{ matrix.deployment == 'helm-operator' }}
+        uses: ./.github/actions/e2e-deploy-vald-helm-operator
+        with:
+          helm_extra_options: "${{ steps.vald_helm_operator.outputs.HELM_EXTRA_OPTIONS }} --set agent.minReplicas=2"
+          valdrelease: ./.github/valdrelease/valdrelease.yaml
+          wait_for_selector: app=vald-lb-gateway
+
+      - name: Deploy Chaos Mesh
+        if: ${{ matrix.chaos }}
```

actionlint (1.7.4)

property "chaos" is not defined in object type {deployment: string; environment: any; mode: string}

(expression)